e2e/cypress/integration/integrations/incoming_webhook/copy_icon_spec.js

```diff
@@ -0,0 +1,65 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [#] indicates a test step (e.g. # Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+describe('Incoming webhook', () => {
+    let testTeam;
+
+    before(() => {
+        // # Set ServiceSettings to expected values
+        const newSettings = {
+            ServiceSettings: {
+                EnableIncomingWebhooks: true,
+            },
+        };
+
+        cy.apiUpdateConfig(newSettings);
+
+        cy.apiInitSetup().then(({team}) => {
+            testTeam = team;
+
+            // # Go to integrations
+            cy.visit(`/${testTeam.name}/integrations`);
+
+            // * Validate that incoming webhooks are enabled
+            cy.get('#incomingWebhooks').should('be.visible');
+        });
+    });
+
+    it('MM-T637 Copy icon for Incoming Webhook URL', () => {
+        const title = 'test-title';
+        const description = 'test-description';
+        const channel = 'Town Square';
+
+        cy.get('#incomingWebhooks').click();
+
+        cy.get('#addIncomingWebhook').click();
```

Comment:
Please make use of [queries from testing-library](https://testing-library.com/docs/dom-testing-library/api-queries) whenever possible. So in this case,
```suggestion
        cy.findByText('Add Incoming Webhook').should('be.visible').click();
```

Same for "Description", "Save", "Setup Successful", and "Done".

Reply:
Is there a way to utilize testing-library queries for targeting an input element associated with a given text? For example, is it possible to access the input element associated with the Title text? I'm thinking something like: `cy.findByText('Title').should('be.visible').type(title);`.

Reply:
I see in the documentation that `findByLabelText` could be used but it still requires the input's id. Is there any benefit to using `cy.findByLabelText('displayName')` over `cy.get('#displayName')`?